src/common/IP/IPManager.cs

```diff
@@ -297,7 +298,7 @@ private void RemoveRoutingRules(CancellationToken cancellationToken, IPAddress[]
                 command: args,
                 logCallback: (line) => _log.Info(line),
                 envVariables: null,
-                timeout: TimeSpan.FromSeconds(10),
+                timeout: TimeSpan.FromSeconds(60),

```

Comment:
💡minor suggestion- if we comment this line with same rationale as mention in description so anyone reading will have relevance latter.

Reply:
@Tatsinnit Thanks for the suggestion, done.